### PR TITLE
Implement design feedback

### LINKED ISSUE #26: Final design pass

### LINKED ISSUE #27: Implement design feedback
See: https://github.com/canonical-web-and-design/buy.ubuntu.com/issues/26

### PR DESCRIPTION
**Pull request by sowasred2012**
## Done

Updated the shopify theme according to design review.

## QA

- Visit https://canonical-staging.myshopify.com/
- See that it complies with the design feedback: ([part 1](https://github.com/canonical-web-and-design/buy.ubuntu.com/issues/26#issuecomment-538359241) & [part 2](https://github.com/canonical-web-and-design/web-squad/issues/1838#issuecomment-542772595))

Fixes #27

### CODE REVIEW

**Comment by sowasred2012** [comment]
/cart:
  - [x] increase bottom border below subtotal to 2px thickness
  - [x] increase padding above footer

**Comment by anasereijo** [comment]
Thanks for doing this @sowasred2012! Pretty much everything looks as expected (design wise) 💯 

Just adding a small change in /cart.

- [x] could you remove the _.00_ from prices? 

- [x] remove USD from each line and add it just at the subtotal

 See my design:

Screenshot 2019-10-17 at 17 32 27

**Comment by matthewpaulthomas** [comment]
> $3,075 USD

A drawback of this layout: the “USD” pushes the number out of alignment with the numbers above. This is especially bad in the usual case — only one item in the cart — because there will be only two numbers, and they’ll be the same number, but unaligned.

A trivial point is that “$3,075 USD” is effectively saying “dollars” twice (since that’s what “D” stands for).

Both could be fixed by rearranging the total as “US $3,075”.

**Comment by matthewpaulthomas** [comment]
> It appears that your cart is currently empty.

Can we drop the “It appears that”, please? To say this, when you just emptied the cart by clicking “Remove”, looks scatter-brained. Sorry I didn‘t notice this earlier.

I’ll review the rest this afternoon.

**Comment by anasereijo** [comment]
@sowasred2012 looks good to me ! thanks

File: src/templates/index.liquid [hunk 1]
```diff
@@ -154,17 +148,31 @@
...
+  </div>
 
-<section class="p-strip--dark" style="background-color: #2c001e;">
-  <div class="row u-vertically-center">
-    <div class="col-8">
-      <h2>{{ settings.home_canonical_heading }}{% if settings.home_canonical_heading_img == true %} <img src="{{ 'canonical-logo.png' | asset_url }}" alt="Canonical">{% endif %}</h2>
-      <p>{{ settings.home_canonical_description }}</p>
-      <p><a href="{{ settings.home_canonical_link | url_param_escape }}" class="p-link--external p-link--inverted">{{ settings.home_canonical_link_text }}</a></p>
+  <div class="row u-equal-height">
+    <div class="col-6 p-card">
```
Comment (solazio) [inline src/templates/index.liquid:157]: are you not supposed to use `p-card` as a child of `col-6`?

File: src/templates/index.liquid [hunk 2]
```diff
@@ -154,17 +148,31 @@
...
+      <h3 class="p-card__title">Kubernetes</h3>
+      <p class="p-card__content u-sv2">Canonical provides training on Kubernetes deployment and operations, with optional extensions for AI/ML development. We can also set up and operate your Kubernetes cluster for you, with in-house experts available at all times to stand up and scale your cluster. And when you're ready, we hand over the keys.</p>
+      <a class="p-link--external" href="https://ubuntu.com/kubernetes">Kubernetes support options</a>
     </div>
 
-    <div class="col-4">
-      <img src="{{ 'canonical-image.png' | asset_url }}" alt="Canonical image">
+    <div class="col-6 p-card">
```
Comment (solazio) [inline src/templates/index.liquid:163]: are you not supposed to use `p-card` as a child of `col-6`?

File: src/assets/style.css [hunk 3]
```diff
@@ -20,85 +24,152 @@
...
+.p-navigation__link--cart:hover {
+  background: #2b2b2b; }
+
+.p-navigation__link--cart-count::before {
+  background-color: #e95420;
+  border-radius: 100%;
+  color: #fff;
+  content: attr(data-item-count);
+  display: block;
+  height: 1.5rem;
```
Comment (solazio) [inline src/assets/style.css:123]: this should be after `font-weight`?

**Review by solazio — COMMENTED** [review]
@sowasred2012 looks good, just a few comments in the code.

File: src/assets/style.css (hunk 3 above)
Comment (sowasred2012) [reply src/assets/style.css:123]: ah yes - I can't alphabet

File: src/templates/index.liquid (hunk 1 above)
Comment (sowasred2012) [reply src/templates/index.liquid:157]: If I make the `p-card` a child, I'd have to add a `height: 100%` to it for the two cards to be equal height. By combining the classes, I get that for free (see also, [the cards on the u.c blog](https://ubuntu.com/blog))

**Review by solazio — APPROVED** [review]
code looks good :+1:

**Comment by matthewpaulthomas** [comment]
- [x] Please reduce the spacing between “Add to cart” and the AWS buy links.

  image

- [x] Please change all “per year” to “/year”, as shown in the recommendations doc and copydocs.

- [x] “Kernel live patch service” has been changed to “Livepatch update service” in some places but not others.

**Comment by sowasred2012** [comment]
> [ ]  Please change all “per year” to “/year”, as shown in the recommendations doc and copydocs.

@matthewpaulthomas Should this include the product cards on the homepage? e.g:

Screenshot 2019-10-18 at 16 05 50

**Comment by matthewpaulthomas** [comment]
> Should this include the product cards on the homepage?

I think so, yes.

**Comment by matthewpaulthomas** [comment]
Last remaining changes from the recommendations doc:

- [x] “In the footer’s “More on commercial support” section, remove all the links except “UA Service Description” (the only one that actually provides more info on commercial support).”

- [x] “On the Physical Server page, replace the IBM z / LinuxONE paragraph with a link, “or _ contact us _ for IBM z / LinuxONE”, below the “Add to Cart” button.”

- [x] “Change tech support details from italics to small print.”

Thanks for all your work on this @sowasred2012.

**Comment by matthewpaulthomas** [comment]
Looks good now, thanks :+1:

**Pull request merged**